tests/test_opera_rtc_s1.py

```diff
@@ -0,0 +1,97 @@
+import json
+import os
+
+import hyp3_sdk
+import pytest
+import requests
+from osgeo import gdal
+
+from hyp3_testing import util
+from hyp3_testing.helpers import archive_tifs, job_tifs
+from hyp3_testing.opera_compare import compare_rtc_hdf5_files, compare_rtc_s1_products
+
+
+gdal.UseExceptions()
+CMR_URL = 'https://cmr.earthdata.nasa.gov/search/granules.umm_json'
+pytestmark = pytest.mark.golden
+
+
+@pytest.mark.nameskip
+def test_golden_submission(comparison_environments):
+    comparison_environments = [comparison_environments[1]]
+    job_name = util.generate_job_name()
+    print(f'Job name: {job_name}')
+
+    testing_parameters = util.render_template('opera_s1_rtc_golden.json.j2', name=job_name)
+    submission_payload = [{k: item[k] for k in ['name', 'job_parameters', 'job_type']} for item in testing_parameters]
+
+    for dir_, api in comparison_environments:
+        dir_.mkdir(parents=True, exist_ok=True)
+
+        hyp3 = hyp3_sdk.HyP3(api, os.environ.get('EARTHDATA_LOGIN_USER'), os.environ.get('EARTHDATA_LOGIN_PASSWORD'))
+        jobs = hyp3.submit_prepared_jobs(submission_payload)
+        request_time = jobs.jobs[0].request_time.isoformat(timespec='seconds')
+        print(f'{dir_.name} request time: {request_time}')
+
+        submission_details = {'name': job_name, 'request_time': request_time}
+        submission_report = dir_ / f'{dir_.name}_submission.json'
+        submission_report.write_text(json.dumps(submission_details))
+
+
+@pytest.mark.timeout(3600)  # 60 minutes as RTC jobs timeout after 1 hour
+@pytest.mark.dependency()
+def test_golden_wait(comparison_environments, job_name, user_id):
+    comparison_environments = [comparison_environments[1]]
```

Comment:
See previous question.
```suggestion
    # TODO: remove this line when we're ready to run jobs in prod
    comparison_environments = [comparison_environments[1]]
```

Reply:
Didn't realize that we're never planning to run jobs in prod for this golden test.